packages/server/src/api/controllers/row/staticFormula.ts

```diff
@@ -156,14 +155,9 @@ export async function finaliseRow(
     contextRows: [enrichedRow],
   })
 
-  const aiEnabled =
-    (await pro.features.isBudibaseAIEnabled()) ||
-    (await pro.features.isAICustomConfigsEnabled())
-  if (aiEnabled) {
-    row = await processAIColumns(table, row, {
-      contextRows: [enrichedRow],
-    })
-  }
+  row = await processAIColumns(table, row, {
```

Comment:
One think is that everybody has access to it, but does it not need to be enabled?

Reply:
Currently it is enabled by default if you have access to it. This PR can't be merged until we put a control in place that has it disabled by default, and requires toggling a setting to activate. I believe Ben and Joe are working on designs for that today.